packages/realtime-api/src/event-handler.ts

```
@@ -0,0 +1,126 @@
+
+  dispatch(eventName: string, event: unknown) {
+    this._log(`dispatch ${eventName} event`);
+
+    const handlers = (
+      this.eventHandlers[eventName] || []
+    ).slice() as Function[];
+    for (const handler of handlers) {
+      handler(eventName, event);
+    }
+
+    const allHandlers = (
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)